packages/client/archive/libp2p/net/package.json.browser.deps

```diff
@@ -65,7 +65,7 @@
     "@ethereumjs/evm": "2.1.0",
     "@ethereumjs/rlp": "5.0.1",
     "@ethereumjs/statemanager": "2.1.0",
-    "@ethereumjs/trie": "6.0.1",
+    "@ethereumjs/mpt": "6.0.1",
```

Comment:
This points to mpt 6.0.1 which will (likely?) not hit a package at all?

Reply:
I have updated everything to 6.2.2